config/j2_analysis.json.empty

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2014-2016 Kartik Kumar (me@kartikkumar.com)
+// Copyright (c) 2014-2016 Abhishek Agrawal (abhishek.agrawal@protonmail.com)
+// Distributed under the MIT License.
+// See accompanying file LICENSE.md or copy at http://opensource.org/licenses/MIT
+
+// Configuration file for D2D "sgp4_scanner" application mode.
+{
+    "mode"                      : "sgp4_scanner",
+
+    // Set path to output database (SQLite).
+    // WARNING: The database file must already exist and be populated with data using the
+    //          "lambert_scanner" mode (data will be store in a table called
+    //          "sgp4_scanner_results")!
```

Comment:
// WARNING: The database file must already exist and be populated with data using the
//                    "lambert_scanner" mode (analysis data will be store in a table called
//                    "j2_analysis_results")!